property/example_test.go

```diff
@@ -31,7 +32,7 @@ import (
 )
 
 // Example to retrieve properties from a single object
-func ExampleCollector_RetrieveOne() {
+func TestExampleCollector_RetrieveOne(*testing.T) {
```

Comment:
These are [Testable Examples](https://go.dev/blog/examples) that are included in the godoc.  Please revert the use of `testing.T` in this file.

The new emptyUpdateSet test can be added to collector_test.go where we using `testing.T`

Reply:
Thanks, will do.